### DISCUSSION TITLE
Creating a file or directory in long paths hitting character limit and cursor jumps back to root path ?

### DISCUSSION THREAD

**Discussion by devspectral** [nvim-tree/nvim-tree.lua#2989] [Q&A] [answered]
Hi! When creating a file/directory in nvim-tree (using "a" key and also happening in renaming a file/folder) with long nested paths, for some reason it is hitting the character limit and will just jump or delete the entire path in the "Create file" box. See screenshot below for reference:

![image](https://github.com/user-attachments/assets/f57e8ab6-52b0-493f-8c0f-de85c7360fff)

Then when it hits the limit (end of the create file bar), it jumps to this.

![image](https://github.com/user-attachments/assets/f6622e6f-7185-4522-a250-556675c60849)

Any thoughts on how to expand the limit? or better yet how to only show the current working directory and not the entire path so that I won't hit that character limit? 

Thank you and I appreciate all the help!

**Comment by alex-courtis** [comment, marked as answer]
It looks like you're using a plugin that provides the input box. nvim-tree should function correctly with it.

Which plugin is it? Can you replicate the issue if that plugin is removed?

**Comment by devspectral** [comment]
Thank you! I just found out that that issue is with dressing.nvim plugin and not nvim-tree. Again thanks!

**Reply by alex-courtis** [reply]
Thanks for getting back.

Is there a workaround we could put on the wiki or in help?

**Reply by devspectral** [reply]
It's a dressing.nvim issue, I tried messing around with the config settings below but the character limit still persists so I just uninstalled the plugin.

```lua
-- These can be integers or a float between 0 and 1 (e.g. 0.4 for 40%)
prefer_width = 40,
width = nil,
-- min_width and max_width can be a list of mixed types.
-- min_width = {20, 0.2} means "the greater of 20 columns or 20% of total"
max_width = { 140, 0.9 },
min_width = { 20, 0.2 },
```

**Reply by devspectral** [reply]
I already posted the issue in the discussions in dressing.nvim, I can file for a bug awaiting for reponse. Thank you for your time.

**Reply by alex-courtis** [reply]
Many thanks @devspectral
